Mathlib/CategoryTheory/Monoidal/Comon_.lean

```diff
@@ -227,10 +230,51 @@ def Mon_op_op_to_Comon : (Mon_ (Cᵒᵖ))ᵒᵖ ⥤ Comon_ C where
 /--
 Comonoid objects are contravariantly equivalent to monoid objects in the opposite category.
 -/
+@[simps]
 def Comon_equiv_Mon_op_op : Comon_ C ≌ (Mon_ (Cᵒᵖ))ᵒᵖ :=
```

Comment:
Should not the capitalization of these definitions be slighly different, like `Comon_EquivMon_OpOp`?

Reply:
I have no idea how the naming rules are mean to apply when there are `_`s in the names. I agree that what I have must be wrong, but your suggestion seems terrible too. :-)

Reply:
Okay, changed to follow the pattern in your suggestion. I guess it is consistent!